StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1115ParameterMustFollowComma.cs

```diff
@@ -319,12 +333,13 @@ private static void AnalyzeParameterList(SyntaxNodeAnalysisContext context, Base
 
         private static bool IsValidTrivia(SyntaxTrivia trivia)
         {
+            if (trivia.IsDirective)
+            {
+                return true;
+            }
```

Comment:
:thought_balloon:  Those would still be reported as SA1123. I don't see any need to make a special consideration for it here as well.